recipes/apache2.rb

```diff
@@ -0,0 +1,53 @@
+#
+# Cookbook Name:: cachethq
+# Recipe:: apache2
+#
+# All rights reserved - Do Not Redistribute
+#
+
+cachethq = node.default['cachethq']
+apache2 = cachethq['apache2']
+
+# somehow apache2 is also installed
+package %w(libapache2-mod-auth-cas apache2)  do
+  action :install
+end
+
+execute "a2enmod auth_cas" do
```

Comment:
need to more consistent here, before this cookbook using bash resource, please use it or make changes on them

```
bash "remove_default_nginx_nginx" do
  code "rm -f /etc/nginx/sites-enabled/default" 
end
```

Reply:
done.